### PR TITLE
SAK-45116 Create a new Meetings tool in Sakai integrated with Microsoft Teams

### PR DESCRIPTION
**Pull request by ropemar**
This PR includes the source code of the new Meetings tool with integration of MS Teams as a videoconferencing provider.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -122,6 +122,7 @@
                 <module>webjars</module>
                 <module>webservices</module>
                 <module>wicket</module>
+                <module>meetings</module>
```
Comment (jesusmmp) [inline pom.xml:125]: Alphabetic order

**Comment by mpellicer** [comment]
Could we target Codacy issues? At least the critical and medium ones, there's many issues that could be fixed easily. Let's spend more time on the code quality please.

**Comment by mpellicer** [comment]
I believe this could be affected by L10N issues, it's fine to fix them now or re-test in multiple languages after the merge.

**Review by mpellicer — APPROVED** [review]
Thank you so much for fixing the Codacy issues, from 367 to 23 Wow! and all minor.

**Comment by ropemar** [comment]
Thank you, @mpellicer :)

**Review by jesusmmp — APPROVED** [review]
Awesome work!

**Comment by mpellicer** [comment]
Thanks Jesus! Let's see how people can test this.... it needs Microsoft credentials so maybe DEV testing is required.

**Review by ern — COMMENTED** [review]
Needs changes

**Comment by mpellicer** [comment]
Fair enough @ern, there's a mirror repository of this project in the contrib area https://github.com/sakaicontrib/meetings, I want to have it private to early adopters or testers as we stated in the T&L presentation. I think this should be a contrib tool for now and for the greater good, I remember other recent projects' like this were called 'experimental' but I suspect they follow different rules.

Right now we're looking for opinions from Microsoft's institutions to define next steps, we're not looking for technical review, for me makes sense to just use the contrib. Shall I revert all the commits from Master or are you willing to do it?

**Comment by ern** [comment]
No need for reverting just will review post merge.

**Pull request merged**